src/com/mambu/apisdk/util/ServiceHelper.java

```diff
@@ -41,6 +41,33 @@
 	 * @return params
 	 */
 	// TODO: when MBU-7042 is fixed - add additional branchId, centreId and centreId filtering params
+	public static ParamsMap makeParamsForGetByCustomView(String customViewKey) {
```

Comment:
This method should not be required if getLoanTransactionsByCustomView(String customViewKey) delegates the request to the existing getLoanTransactionsByCustomView(String customViewKey, String offset, String limit) method.